.github/workflows/enteprise.yml

```diff
@@ -28,7 +28,7 @@ jobs:
   docker:
     name: Test ${{ matrix.role.name }} role on ${{ matrix.molecule_distro.image }}
     runs-on: ubuntu-latest
-    if: github.repository_owner == 'Alfresco'
+    if: ${{ ! github.event.pull_request.head.repo.fork || github.event_name == 'push' }}
```

Comment:
That would fail on forked repo outside of the org (when pushing to forked master branch without a PR) and be skipped in case of PR from a fork within the Alfresco org.

Reply:
the problem here is just that we don't want to run this workflow at all when secrets are not available

the previous `if: github.repository_owner == 'Alfresco'` was not working either since I switched to `pull_request` (not sure 100% but I've seen it failing on this PR)

docs says we cannot directly reference secrets inside job conditionals, but I have an idea I want to try, in another PR

Reply:
https://github.com/Alfresco/alfresco-ansible-deployment/pull/823